### ISSUE TITLE
let list items link to news details

### ISSUE THREAD

**Issue by testbird** [Flow-Community/Community.News#9]
I don't see any links in the news list that pointing to the news detail page, is that correct?
How does Community.News handle this?

I see the lelesys package
http://www.lelesys.com/en/technology/about-typo3-neos/neos-packages/news-package-extension-for-typo3-neos.html
renders "read more" links:
http://news-for-neos.demo.lelesys.de/news.html

And the example in the Neos Docs makes the entire teaser text a link:
https://neos.readthedocs.org/en/stable/CreatingASite/RenderingCustomMarkup/Templating.html?highlight=blog

The latter has the benefit of not needing any translation. It may be more easy for the visitor to see that there is more by appending some " ..." to the teaser text link, if there is actually is more content.

**Comment by dimaip** [comment]
Wow that's like really weird, indeed there's no link! I never used this package in production myself, maybe @Weissheiten knows why it's this way.
I would wrap the whole object with a link, like I do here: http://sfi.ru/ The bigger clickable area the better.

**Comment by Weissheiten** [comment]
Yep, this should be in the standard template too, chances are great you'll exchange the basic template for your own anyways though - nevertheless I agree this should be fixed in the base version.
